app/controllers/papers_controller.rb

```diff
@@ -16,6 +16,7 @@ def show
   # GET /papers/new
   def new
     @paper = Paper.new
+    5.times { @paper.authors.build }
```

Comment:
This is a hack that we should probably mark with some sort of comment like TODO or SHAME. We'll be able to remove this when we have a real front-end.